### DISCUSSION TITLE
Does this need a clean Arch install?

### DISCUSSION THREAD

**Discussion by DoomPenguin9** [prasanthrangan/hyprdots#1984] [Q&A]
My current Arch install is over 3 years old, and I don't want to lose anything. Does hyprdots require a fresh Arch install, or can I install it over an existent Arch/Hyprland setup? All the tutorials I've seen use a fresh installation.

**Comment by kRHYME7** [comment]
Test it on a new user

**Reply by tobin-rieger-cameron** [reply]
whoa that works??
